### ISSUE TITLE
option for light background and setting to specify colors

### ISSUE THREAD

**Issue by solsticedhiver** [Kiwi/clyde#25]
I use I white background for my terminal and clyde looks ugly.

An option in the conf file (better if it is ~/.clyde.conf) to select a light background color scheme. would be nice

And/Or the possiblity to specify colors in the conf file. to make your own color scheme

**Comment by nogweii** [comment]
I would like to add that using /etc/pacmand.d/color.conf as another possible source of color related information would be great. `pacman-color`, while it hasn't been updated in ever, used (and created it) this file to store the color configuration. More compatibilty with other tools, the better for clyde, I think.

**Comment by mk12** [comment]
I agree with evaryont, please let us use /etc/pacman.d/color.conf for colour, pacman-color looks a lot better than clyde to me, there are more colours.

**Comment by dhave** [comment]
+1 for option to control clyde's colors. On my monitor, the highlight for installed packages shows up as a ghoulish orange. Not a show-stopper by any means, but this would be a welcome improvement. Thanks!

**Comment by vieira** [comment]
Some recent changes made things worse. Now the package names are white (even for people with a white background, as me) and it's unreadable.
